MIT.md

```diff
@@ -0,0 +1,11 @@
+## Copyright 2021, [VAHAN KHACHVANKIAN]
+
+###### Please delete this line and the next one
+
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
- [ ] [OPTIONAL]: Great job adding an MIT license file, but in order to keep your repo more professional, you should delete these 2 lines, as suggested by the own file.

Reply:
Done.